app/views/search/_doi.html.erb

```diff
@@ -1,22 +1,9 @@
 <% return unless @enhanced_query[:doi].present? %>
 
-<% json = FactDoi.new.info(@enhanced_query[:doi]) %>
-<% return unless json.present? %>
+<% data_url = "/doi?doi=#{URI.encode_www_form_component(@enhanced_query[:doi])}" %>
 
-<div id="doi-fact" class="fact">
-  <div class="panel panel-success">
-    <div class="panel-heading">
-      <%= json[:title] %>
-    </div>
-
-    <div class="panel-body">
-      <ul>
-        <li><%= json[:publisher] %></li>
-      </ul>
-    </div>
-
-    <div class="panel-footer">
-      <a href="#">Yes this</a> -- <a href="#">Not relevant to my search</a>
-    </div>
-  </div>
+<div id="doi-fact"
```

Comment:
It looks like we have repeated `id` values for this structure, between the search partials and the fact templates? Can we remove one, or should we change them to not be identical?

(I'm making the same comment on each of the `doi-fact` elements, but I think all four are doubled)

Reply:
Are you seeing that in the rendered HTML? I thought stimulus was replacing the content in a way that there was only ever one of these ids live at a time... but I may have misinterpreted (or just coded it wrong even interpreting it correctly lol)

Reply:
Ah, I found it. Definitely doubling up for dois (and likely others). PMID I had checked and it doesn't double because I typod the first ID to be wrong lol. Yup, this needs fixing on all of them. Good catch!